releasenotes/notes/nogevent-6f2892cb412f987f.yaml

```diff
@@ -0,0 +1,9 @@
+---
+deprecations:
+  - |
+    internal: `DD_GEVENT_PATCH_ALL` is deprecated and will be removed in the next major verison. Gevent compatibility is now automatic
```

Comment:
should we add a deprecation warning if we detect that the variable is still set to let the users know the variable is no longer needed? @brettlangdon WDYT?